apps/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,12 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
🔤

Reply:
I guess expo is going to keep doing this later if I roll back now, might as well keep it as is.